client/web/src/user/settings/auth/ExternalAccountsSignIn.tsx

```diff
@@ -95,6 +116,33 @@ const getNormalizedAccount = (
                     }
                 }
                 break
+            case 'SAML':
+                {
+                    const samlExternalData = accountExternalData as SamlExternalData
+                    const data = samlExternalData.Assertions[0].AttributeStatement.Attributes
+                    let nickname = ''
+
+                    for (let index = 0, length = data.length; index < length; index += 1) {
+                        const item = data[index]
+                        if (data[index].Values && item.Name) {
```

Comment:
I think we prefer [`for ... of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) whenever possible.

I think this should work:

```suggestion
                    for (const item of data) {
                        if (item.Values && item.Name) {
```

Reply:
+1